pkg/utils/resource.go

```diff
@@ -61,3 +62,25 @@ func GetResourceCount(resourceMapping map[string][]output.ResourceConfig) (count
 	}
 	return
 }
+
+// UpdateResourceConfigs adds a resource of given type if it is not present in allResources
+func UpdateResourceConfigs(resourceType string, resources []output.ResourceConfig, allResources output.AllResourceConfigs) {
```

Comment:
Hey @patilpankaj212 , 
Does it make sense to have this function and other functions in this file to be moved to the output package itself. Come to think of it these are not `utils` in real sense but specific use cases related to the `AllResourcesConfig` in the output package.